docs/source/conf.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/stable/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+import os
+import sys
+import sphinx_rtd_theme
+
+module_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),'../../')
+sys.path.insert(0, module_path)
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'nabs'
+copyright = '2018, SLAC National Accelerator Laboratory'
+author = 'SLAC National Accelerator Laboratory'
+
+# The short X.Y version
+import nabs
+version = nabs.__version__
+# The full version, including alpha/beta/rc tags
+release = nabs.__version__
+
+
+# -- General configuration ---------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.viewcode',
+    'sphinx.ext.githubpages',
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
```

Comment:
Add line anywhere in this file:
```
autosummary_generate = True
```
So you don't need to call `sphinx-autogen` every time